### ISSUE TITLE
Playback Error iOS. 

### ISSUE THREAD

**Issue by litris** [jellyfin/jellyfin-ios#207]
I've read a lot about it without any solution.

The films are reproduced without a problem at Xiaomi Tv box and Infuse on an iPhone but not on an iPhone.

Before this didn't happen, I had every similar thing on my iPhone.

It's .mkv at 720p and AC3.

Any solutions?

**Comment by anthonylavado** [comment]
We'll need some more information. What version of Jellyfin are you running? What language is the operating system in?

**Comment by litris** [comment]
> We'll need some more information. What version of Jellyfin are you running? What language is the operating system in?

It is not necessary.
It is a Bug known to all.
It will be arrested in the next version I guess.

**Comment by anthonylavado** [comment]
The reason I ask is because there is a bug that would be fixed in the next version of the server - however this bug only occurs if the server is not running in English, and only if the client is Safari (so iOS through this app, and macOS).

I'll keep this issue open until that release happens, but that's why I was asking for more information 😕

**Comment by anthonylavado** [comment]
I see you already asked and got an answer here:
https://github.com/jellyfin/jellyfin/issues/5128

It would have been better to just keep it in one issue… ☹️

**Comment by litris** [comment]
> I see you already asked and got an answer here:
> [jellyfin/jellyfin#5128](https://github.com/jellyfin/jellyfin/issues/5128)
> 
> It would have been better to just keep it in one issue… ☹️

I opened it from the app, but the message was not shown in Gihub.

I closed the solution and created one in github directly.

But days later, it appeared, it's another mistake of the solution.

It's not my fault I just pulled a thread and the iOs app didn't turn it up.

Days later he uploaded it is pretty weird.

That's why there's two strings.

This shouldn't be here.

The right one is the other one that was instantly created in Github.
